examples/llama.py

```diff
@@ -344,6 +349,9 @@ def main(string: str):
   parser.add_argument("--shard", type=int, default=1, help="number of devices to load the weights to")
 
   args = parser.parse_args()
+  if not args.model:
```

Comment:
i think it's better to only do this only if `gen==tiny`, otherwise this silently overwrites the default `gen==1`

Reply:
Ok, was going off geohot comment from Discord:
"can someone make llama autodownload small llama weights if you don't pass in anything?"

Set to =="tiny" only now.